index.html

```diff
@@ -718,6 +344,19 @@
 
   console.log(100, "WebGL2 canvas created successfully");
 
+  //
+  //   Load shader source code
+  //
+  async function loadShaderSource(url) {
+    const response = await fetch(url);
+    return await response.text();
```

Comment:
```suggestion
    const response = await fetch(url);
    const text = await response.text();
    return text;
```

A bit easier for the non-JavaScript experts
<---